packages/wdio-cli/src/launcher.js

```diff
@@ -296,14 +302,20 @@ class Launcher {
             return
         }
 
-        if (this.exitCode === 0) {
-            return this.resolve(this.exitCode)
+        if (passed) {
+            return process.nextTick(() => {
+                this.interface.updateView()
+                setTimeout(() => this.resolve(this.exitCode), 100)
```

Comment:
Perhaps updateView should return a promise so the setTimeout isn't needed?

Note: This might be unrelated to the PR but I thought it was worth mentioning

Reply:
no, this is only to give the cli a bit time to render the last update view change